uploader.json

```diff
@@ -1775,6 +1782,14 @@
             ],
             "symbolic_name": "vdsm_log"
```

Comment:
Please refer https://github.com/RedHatInsights/insights-frontend-assets/pull/35/ for pattern related to `vdsm_log`

Reply:
#35 is for `vdsm_log`, it's not related to the `vdsm.conf`